zarnitsa/DataAugmenterExternally.py

```diff
@@ -3,108 +3,130 @@
+            "gamma": dict(
+                shape=kwargs["shape"], scale=kwargs["scale"], size=size
+            ),
+            "geometric": dict(p=kwargs["p"], size=size),
+            "gumbel": dict(
+                loc=kwargs["loc"], scale=kwargs["scale"], size=size
+            ),
+            "hypergeometric": dict(
+                ngood=kwargs["ngood"],
+                nbad=kwargs["nbad"],
+                nsample=kwargs["nsample"],
+                size=size,
+            ),
+            "laplace": dict(
+                loc=kwargs["loc"], scale=kwargs["scale"], size=size
+            ),
+            "logistic": dict(
+                loc=kwargs["loc"], scale=kwargs["scale"], size=size
+            ),
+            "lognormal": dict(
+                mean=kwargs["mean"], sigma=kwargs["sigma"], size=size
+            ),
+            "logseries": dict(p=kwargs["p"], size=size),
+            "multinomial": dict(
+                n=kwargs["n"], pvals=kwargs["pvals"], size=size
+            ),
+            "multivariate_norma": dict(
+                mean=kwargs["mean"], cov=kwargs["cov"], size=size
+            ),
+            "negative_binomial": dict(n=kwargs["n"], p=kwargs["p"], size=size),
+            "noncentral_chisquare": dict(
+                df=kwargs["df"], nonc=kwargs["nonc"], size=size
+            ),
+            "noncentral_f": dict(
+                dfnum=kwargs["dfnum"],
+                dfden=kwargs["dfden"],
+                nonc=kwargs["nonc"],
+                size=size,
+            ),
+            "normal": dict(
+                loc=kwargs["loc"], scale=kwargs["scale"], size=size
+            ),
+            "pareto": dict(a=kwargs["a"], size=size),
+            "poisson": dict(lam=kwargs["lam"], size=size),
+            "power": dict(a=kwargs["a"], size=size),
+            "rayleigh": dict(scale=kwargs["scale"], size=size),
+            "standard_cauchy": dict(size=size),
+            "standard_exponential": dict(size=size),
+            "standard_gamma": dict(shape=kwargs["shape"], size=size),
+            "standard_normal": dict(size=size),
+            "standard_t": dict(df=kwargs["df"], size=size),
+            "triangular": dict(
+                left=kwargs["left"],
+                mode=kwargs["mode"],
+                right=kwargs["right"],
+                size=size,
+            ),
+            "uniform": dict(low=kwargs["low"], high=kwargs["high"], size=size),
+            "vonmises": dict(
+                mu=kwargs["mu"], kappa=kwargs["kappa"], size=size
+            ),
+            "wald": dict(
+                mean=kwargs["mean"], scale=kwargs["scale"], size=size
+            ),
+            "weibull": dict(a=kwargs["a"], size=size),
+            "zipf": dict(a=kwargs["a"], size=size),
+        }
+        np_func = getattr(np.random, aug_type, None)
+        if np_func:
+            np_func(**np_distributions[aug_type])
```

Comment:
cool! i should learn from u a lot!

Reply:
this one can reduce flexibility at the moment, but it can be easily improved to handle any function, not only numpy methods

Reply:
well, will see )